### ISSUE TITLE
Change base.yml main keys to be package manager (or other identifiable binary) names rather than repo-tags

### ISSUE THREAD

**Issue by nishakm** [vmware/tern#61]
The repo-tag is a container registry construct and could change depending on the registry. So look up the base commands based on the package manager rather than the repo tag.

Eg:
```
vmware/photon:
  latest: '1.0'
  tags:
    '1.0':
      shell: '/usr/bin/bash'
      names:
```
to

```
tdnf:
  repo: 'vmware/photon'
  tags: ['latest', '1.0']
  shell: '/usr/bin/bash'
  names:
```
This is to support #51

**Comment by nishakm** [comment]
@snmpboy Are you free to work on this issue? It's on a deadline so no pressure if you cannot commit.

**Comment by snmpboy** [comment]
@nishakm Kind of slammed at the moment.  Working and studying for https://www.cisco.com/c/en/us/training-events/training-certifications/exams/current-list/300-560-npdev.html

**Comment by nishakm** [comment]
@snmpboy no worries! Good luck with the networking study - that's intense!

**Comment by nishakm** [comment]
repo and tag may not be a good fit for this as they are representative of container distribution and not the contents. Values can be stored in the database instead.

**Comment by nishakm** [comment]
Closed in #67
